.github/workflows/ioc-images.yml

```diff
@@ -0,0 +1,38 @@
+name: Build image
+on:
+  workflow_call:
+    secrets:
+      token:
+        required: true
+
+jobs:
+  build_and_push:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    env:
+      REGISTRY: ghcr.io/${{ github.repository_owner }}
+      SOURCE: https://github.com/${{ github.repository }}
+      TAG: ${{ github.ref_name }}
```

Comment:
We should document that this must be specified in the `docker-compose.yml` image name so that the tagged image is pushed. Otherwise, only the `latest` image tag will be used.

Also change the template in the README.md to use it.